<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>735</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CowPlayer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="video_player" name="videoPlayer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Phonon::SeekSlider" name="seekSlider"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="playButton">
        <property name="text">
         <string>Pause</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Phonon::VolumeSlider" name="volumeSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="baseSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionShow_program_list"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionFullscreen"/>
    <addaction name="actionShow_pieces"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_CowPlayer"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionShow_program_list">
   <property name="text">
    <string>Show program list...</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
  </action>
  <action name="actionAbout_CowPlayer">
   <property name="text">
    <string>About CowPlayer...</string>
   </property>
  </action>
  <action name="actionShow_pieces">
   <property name="text">
    <string>Show pieces...</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
  <customwidget>
   <class>video_player</class>
   <extends>QWidget</extends>
   <header>video_player.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
